### PR TITLE
{geo}[foss/2022a] rgdal v1.6-6

### PR DESCRIPTION
**Pull request by terjekv**
(created using `eb --new-pr`)

### CODE REVIEW

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/r/rgdal/rgdal-1.6-6-foss-2022a-R-4.2.1.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# This easyconfig was created by Simon Branford of the BEAR Software team at the University of Birmingham.
+easyblock = 'RPackage'
+
+name = 'rgdal'
+version = '1.6-6'
+versionsuffix = '-R-%(rver)s'
```
[the remark this answers is not in the document]
Comment (terjekv) [reply easybuild/easyconfigs/r/rgdal/rgdal-1.6-6-foss-2022a-R-4.2.1.eb:6]: Ah, yeah, removing. Thanks.

File: easybuild/easyconfigs/r/rgdal/rgdal-1.6-6-foss-2022a-R-4.2.1.eb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    'https://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
+]
+sources = ['%(name)s_%(version)s.tar.gz']
+checksums = ['d742d7aadfc004771e61cac28a1faffeb4dbda981dea19115be11c541087399a']
+
+dependencies = [
+    ('R', '4.2.1'),
+    ('GDAL', '3.5.0'),
+]
+
```
[the remark this answers is not in the document]
Comment (terjekv) [reply easybuild/easyconfigs/r/rgdal/rgdal-1.6-6-foss-2022a-R-4.2.1.eb:26]: Will do. Sorry about this.

File: easybuild/easyconfigs/r/rgdal/rgdal-1.6-6-foss-2022a-R-4.2.1.eb (hunk 1 above)
[the remark this answers is not in the document]
Comment (terjekv) [reply easybuild/easyconfigs/r/rgdal/rgdal-1.6-6-foss-2022a-R-4.2.1.eb:6]: Fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/17798/commits/2c8afe8b366004a39a8d286b1d33f9de6d459521

File: easybuild/easyconfigs/r/rgdal/rgdal-1.6-6-foss-2022a-R-4.2.1.eb (hunk 2 above)
[the remark this answers is not in the document]
Comment (terjekv) [reply easybuild/easyconfigs/r/rgdal/rgdal-1.6-6-foss-2022a-R-4.2.1.eb:26]: Fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/17798/commits/2c8afe8b366004a39a8d286b1d33f9de6d459521

**Comment by terjekv** [comment]
Is there a way to rename the easyconfig itself, or should I make a PR with the new correct name?

**Comment by verdurin** [comment]
Looks like you've already done it, thanks.

**Comment by terjekv** [comment]
> Looks like you've already done it, thanks.

I figured I could try the `:` trick with `--update-pr`. I just wasn't sure if it'd work on the only file in the commit. :)

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
@boegelbot please test @ jsc-zen2

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @terjekv!

**Pull request merged**